rstcheck.py

```diff
@@ -875,10 +881,40 @@ def enable_sphinx_if_possible():
         yield
 
 
+def match_file(filename):
+    """Return True if file is okay for modifying/recursing."""
+    base_name = os.path.basename(filename)
+
+    if base_name.startswith('.'):
```

Comment:
This is to ignore hidden files, right? (Not sure why they should be ignored, but that's OK, I don't have any :)